### DISCUSSION TITLE
Changing localhost for IP address

### DISCUSSION THREAD

**Discussion by jamodio** [nasa/openmct#5736] [Community Q&A] [answered]
Hi There,

how I can change localhost for an IP address so I can run the openMCT dev server on a headless machine and access it from my desktop browser ?

TIA
Jorge

**Comment by jamodio** [comment]
Never mind, found the definition on app.js :-)

// Defaults
options.port = options.port || options.p || 8080;
options.host = options.host || 'localhost';   <-------------------
options.directory = options.directory || options.D || '.';

Warm Regards
Jorge

**Comment by unlikelyzero** [comment, marked as answer]
@jamodio this will change in the very near future. We do not recommend running app.js in "production" and will be providing some 
additional documentation here https://github.com/nasa/openmct/issues/5812
